typify-impl/src/lib.rs

```diff
@@ -205,21 +205,28 @@ pub struct TypeSpaceSettings {
 /// Per-type modifications.
 #[derive(Debug, Default, Clone)]
 pub struct TypeSpacePatch {
+    /// Replacement name
```

Comment:
as these fields are not `pub`, I don't think this external documentation is helpful to consumers.

Reply:
I found it confusing that the "replacement name" field was different for each of these three, and there isnt a good place to describe that. Anyway, removed.